source/locale/zh_CN/LC_MESSAGES/reference/generated/optuna.pruners.HyperbandPruner.po

```diff
@@ -145,35 +165,45 @@ msgid ""
 " the number of epochs for neural networks). When this argument is "
 "\"auto\", the maximum resource is estimated according to the completed "
 "trials. The default value of this argument is \"auto\"."
-msgstr ""
+msgstr "一个用于指定分配给 trial 的最大资源的参数."
```

Comment:
I notice a similar paragraph above this, from line 146

Reply:
Line 146 has `  .. note::    ` so it's not translated, similar to the untranslated example above